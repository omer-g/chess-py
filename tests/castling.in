e2 e4
e7 e5
f1 c4
f8 c5
g1 f3
g8 f6
e1 g1
e8 g8
h2 h3
h7 h6
0