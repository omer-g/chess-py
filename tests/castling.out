
        ##########################
        ######## chess-py ########
        ##########################

                    #
                   ###
                  ####
                 #### ##
                #### ####
                 ## ###
                  ####
               ##########
                  ####
                  ####
                  ####
                  ####
             ##############
             ##############

move: 'e2 e4' revert: 'r' exit: '0'

   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|bP|bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |  |  |  |  |
4 |  |  |  |  |  |  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|wP|wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|bP|bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |  |  |  |  |
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |  |
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|  |wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|  |bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|  |wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|  |bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|  |  |wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|  |  |bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |bN|  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|  |  |wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|  |  |bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |bN|  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|  |wR|wK|  |

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|  |bR|bK|  |
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |bN|  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|  |wR|wK|  |

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|  |bR|bK|  |
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |bN|  |  |
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |wP|
2 |wP|wP|wP|wP|  |wP|wP|  |
1 |wR|wN|wB|wQ|  |wR|wK|  |

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|  |bR|bK|  |
7 |bP|bP|bP|bP|  |bP|bP|  |
6 |  |  |  |  |  |bN|  |bP|
5 |  |  |bB|  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |wN|  |wP|
2 |wP|wP|wP|wP|  |wP|wP|  |
1 |wR|wN|wB|wQ|  |wR|wK|  |

enter move
